libs/isograph-react/src/tests/__isograph/Query/subquery/entrypoint.ts

```diff
@@ -0,0 +1,65 @@
+import type {IsographEntrypoint, NormalizationAst, RefetchQueryNormalizationArtifactWrapper} from '@isograph/react';
+import {Query__subquery__param} from './param_type';
+import {Query__subquery__output_type} from './output_type';
+import readerResolver from './resolver_reader';
+const nestedRefetchQueries: RefetchQueryNormalizationArtifactWrapper[] = [];
+
+const queryText = 'query subquery ($id: ID!) {\
+  query {\
+    node____id___v_id: node(id: $id) {\
+      __typename,\
+      id,\
+    },\
+  },\
+}';
+
+const normalizationAst: NormalizationAst = [
+  {
+    kind: "Linked",
+    fieldName: "query",
+    arguments: null,
+    concreteType: "Query",
+    selections: [
+      {
+        kind: "Linked",
+        fieldName: "node",
+        arguments: [
+          [
+            "id",
+            { kind: "Variable", name: "id" },
+          ],
+        ],
+        concreteType: null,
```

Comment:
we should have a follow up to remove the `__typename` selection from below. It only really needs to show up in the query text. In the query text, `__typename` will never be aliased, so we can do `selection.concreteType ?? data.__typename`.

#178. Feel free to do it as part of this PR, it should be a relatively small change. But also, happy to have it be a follow up or left for someone else.